app/controllers/api/v1/short_form_controller.rb

```diff
@@ -37,112 +38,104 @@ def application_api_params
             :status,
             :numberOfDependents,
             :formMetadata,
-            {
```

Comment:
this was due to a rubocop error. Style/BracesAroundHashParameters defaults to no_braces, and we don't have anything in rubocop.yml that says otherwise.